README.md

```diff
@@ -1,41 +1,36 @@
 # Raspberry Pi Docker Swarm instructions
 
-## Initial SD card operating system installation
-
-1. Download [Raspberry Pi Imager](https://www.raspberrypi.com/software/)
-2. Start up Raspberry Pi Imager and use "CHOOSE DEVICE" to select your
-   Raspberry Pi model
-3. Select "CHOOSE OS" and navigate to
-   "Other general-purpose OS" -> Ubuntu -> "Ubuntu Server 24.04.1 LTS (64-bit)"
-
-   NOTE: If you are using a Raspberry Pi 2 or below, or a Raspberry Pi Zero,
-   you'll need to pick the 32-bit version instead, and some apps may take some
-   effort to get working
-4. Insert your SD Card into your computer
-5. Select "CHOOSE STORAGE" and select the SD Card you've just inserted. Ensure
-   you have the correct device, as it'll be completely wiped and you'll lose
-   anything on it
-6. Select "NEXT"
-7. When asked "Would you like to apply OS customisation settings?", Select "NO"
-8. You'll now receive your final prompt to confirm this is the correct storage
-   device to wipe. Check this, and then select "YES"
-9. When prompted, remove the SD card and close Raspberry Pi Imager
-
-## Customising operating system configuration
-
-1. Insert the SD card you've just freshly flashed with Ubuntu Server
+## Preparing a Generic SD Card
+
+See [Preparing SD Card](./docs/preparing_generic_sd_card.md) to setup a generic SD Card
+for a swarm PI, unless someone else has prepared one for you.
+
+## Customising SD Card for you Site
+
+1. Insert the Generic SD card from the step above into a card reader
+   connected to your computer.
 2. Your computer should detect the card and you should be able to find a new
    drive in your file manager. On MacOS this will be called "system-boot"
-3. Copy all of the files from this repository's `cloud-init` directory into the
-   top directory of the SD card
-4. Eject the SD card from your computer
+3. Open the file `user-data` in a text editor and find the line
+   `hostname: raspberrypi`. Change the word `raspberrypi` to a hostname
+   likely to be unique on your local network. If you're setting up a site
+   called `mysite`, we recommend `mysite1` as a hostname (and this will
+   be used as an example in following instructions).
+4. Open up the file `network-config` in a text editor. Find the section
+   with the line `access-points:`. Underneath that, any number of WiFi
```

Comment:
I didn't mention Wifi credentials on the understanding we were expecting these to be connected via ethernet instead. Happy for the change, but unsure if it's strictly necessary here, or just going to add unnecessary effort or confusion?

Reply:
Fair point. Interestingly I default to wifi during setup because I wanted to play with the Pi in my room, and not take it downstairs to where the router is. I was more assuming that using ethernet was an optimisation we'd do later. Wifi might be particularly needed during shared workshops, and during the setup phase in a community facility we don't control well.

I propose that I make a change here, describing this as an optional step if you aren't going to use ethernet.

Reply:
Sounds good to me!